packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit_webview_widget.dart

```diff
@@ -78,15 +78,19 @@ class WebKitWebViewPlatformController extends WebViewPlatformController {
     required this.callbacksHandler,
     required this.javascriptChannelRegistry,
     WKWebViewConfiguration? configuration,
-    @visibleForTesting this.webViewProxy = const WebViewProxy(),
+    @visibleForTesting this.webViewProxy = const WebViewWidgetProxy(),
   }) : super(callbacksHandler) {
     _setCreationParams(
       creationParams,
-      configuration: configuration ?? WKWebViewConfiguration(),
-    ).then((_) => _initializationCompleter.complete());
```

Comment:
This method doesn't actually wait for any `Future` and  `_initializationCompleter` is not used. I missed this in the PR before this one.